flink-cdc-e2e-tests/flink-cdc-pipeline-e2e-tests/pom.xml

```diff
@@ -261,6 +275,33 @@ limitations under the License.
                             <outputDirectory>${project.build.directory}/dependencies
                             </outputDirectory>
                         </artifactItem>
+                        <artifactItem>
+                            <groupId>org.apache.flink</groupId>
+                            <artifactId>flink-cdc-pipeline-connector-paimon</artifactId>
+                            <version>${project.version}</version>
+                            <destFileName>paimon-cdc-pipeline-connector.jar</destFileName>
+                            <type>jar</type>
+                            <outputDirectory>${project.build.directory}/dependencies
+                            </outputDirectory>
+                        </artifactItem>
+                        <artifactItem>
+                            <groupId>org.apache.flink</groupId>
+                            <artifactId>flink-shaded-hadoop-2-uber</artifactId>
+                            <version>2.8.3-10.0</version>
+                            <destFileName>flink-shade-hadoop.jar</destFileName>
+                            <type>jar</type>
+                            <outputDirectory>${project.build.directory}/dependencies
+                            </outputDirectory>
+                        </artifactItem>
+                        <artifactItem>
+                            <groupId>org.apache.paimon</groupId>
+                            <artifactId>paimon-flink-${flink.major.version}</artifactId>
```

Comment:
Flink version might vary depending on E2e runtime arguments. Maybe we need to declare multiple Flink version and select corresponding jars dynamically (just like JDBC connector in Source E2e tests)?

Reply:
Addressed it.